app/jobs/channelJobs.js

```diff
@@ -388,7 +390,158 @@ var job = {
             });
             return { success: 0, message: ex };
         }
-    }
+    },
+  
+    //jira method start
+    AddJiraConfigs: async function (request) {
+        if (request) {
+            const transaction = await db.sequelize.transaction();
+            let channel;
+            try {
+                var jira = await models.ChannelConfigs.findOne({
+                    where: {
+                        communication_channel_id: "Jira"
+                    }
+                });
+
+                var password = util.encrypt(request.emailParameter.password);
```

Comment:
2 steps can be changed to one step. no need to create new var
 request.emailParameter.password = util.encrypt(request.emailParameter.password);